weaviate/gql/get.py

```diff
@@ -610,6 +619,546 @@ def with_near_image(self, content: dict, encode: bool = True) -> "GetBuilder":
         self._contains_filter = True
         return self
 
+    def with_near_audio(self, content: dict, encode: bool = True) -> "GetBuilder":
+        """
+        Set `nearAudio` filter.
+
+        Parameters
+        ----------
+        content : dict
+            The content of the `nearObject` filter to set. See examples below.
+        encode : bool, optional
```

Comment:
can't we autodetect that?

Reply:
yes and know, we can rely either on a fact that if someone passes `{filename}.{suffix}` we detect that and try to load the file, also we can do the other way round if someone **doesn't** pass a `base64 encoded string` that we try to treat it as a file

Reply:
we could accept either:
- a path object
- an open file
- a string, which must be base64 encoded

and distinguish with isinstance?

Reply:
too much work now